include/openmc/geometry_aux.h

```diff
@@ -63,9 +64,12 @@ int32_t find_root_universe();
 
 //==============================================================================
 //! Populate all data structures needed for distribcells.
+//! \param user_distribcells A set of cell indices to create distribcell data
+//!   structures for regardless of whether or not they are part of a tally
+//!   filter.
 //==============================================================================
 
-void prepare_distribcell();
+void prepare_distribcell(const std::unordered_set<int32_t>* user_distribcells = nullptr);
```

Comment:
Would it make more sense to just have this accept a `vector` rather than an `unordered_set`?

Reply:
Mm yeah I'd say it would. Wasn't thinking there. Thanks! I'll update that.

Do you think a `std::vector` or an `openmc::vector`? I'm leaning `std::vector` since it'll likely be passed externally and requiring use of our custom data structure in an external app might not be well-received if that vector is also being used elsewhere to track the cells.

Reply:
Yes, for a user-facing API definitely use `std::vector`